sdk/python/kfp/_credentials.py

```diff
@@ -12,16 +12,24 @@
 # See the License for the specific language governing permissions and
```

Comment:
Can this file be moved elsewhere? E.g. into a `client` or `auth` package ? So it's not in the root kfp folder.

Reply:
Ack, moved to `kfp.auth`